packages/test-utils/src/wrapper.js

```diff
@@ -582,3 +587,23 @@ export default class Wrapper implements BaseWrapper {
     this.element.dispatchEvent(event)
   }
 }
+
+export const enableAutoDestroy = (hook: Function) => {
+  if (Wrapper._instances) {
+    throwError('enableAutoDestroy cannot be called more than once')
+  }
+
+  Wrapper._instances = []
+
+  hook(() => {
+    const { _instances } = Wrapper
+    if (!_instances) {
+      return
```

Comment:
should we throw an error here?

Reply:
This would only happen if something manually overwrote `Wrapper._instances` to some falsey value? I think that warrants an error with a friendly (yet stern) message :wink: